app/src/main/java/tech/httptoolkit/android/HttpToolkitApplication.kt

```diff
@@ -263,17 +276,21 @@ class HttpToolkitApplication : Application() {
 }
 
 private fun wasInstalledFromStore(context: Context): Boolean {
-    return context.packageManager.getInstallerPackageName(context.packageName) != null
+    return if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
+        context.packageManager.getInstallSourceInfo(context.packageName).installingPackageName != null
+    } else {
+        context.packageManager.getInstallerPackageName(context.packageName) != null
+    }
 }
 
 private fun firstInstallTime(context: Context): Long {
     return context.packageManager.getPackageInfo(context.packageName, 0).firstInstallTime
 }
 
 private data class GithubRelease(
-    val tag_name: String?,
+    val tagName: String?,
     val name: String?,
-    val published_at: String
+    val publishedAt: String,
```

Comment:
Does this work for you? On my machine this fails with:

```
Unable to instantiate GithubRelease:No argument provided for a required parameter: parameter #0 tagName of fun `<init>`(kotlin.String?, kotlin.String?, kotlin.String): tech.httptoolkit.android.GithubRelease
No argument provided for a required parameter: parameter #0 tagName of fun `<init>`(kotlin.String?, kotlin.String?, kotlin.String): tech.httptoolkit.android.GithubRelease
```

I think the names here need to match the actual JSON field keys.